include/umf/providers/provider_cuda.h

```diff
@@ -14,12 +14,44 @@
 extern "C" {
 #endif
 
-/// @brief CUDA Memory Provider settings struct
-typedef struct cuda_memory_provider_params_t {
-    void *cuda_context_handle;         ///< Handle to the CUDA context
-    int cuda_device_handle;            ///< Handle to the CUDA device
-    umf_usm_memory_type_t memory_type; ///< Allocation memory type
-} cuda_memory_provider_params_t;
+struct umf_cuda_memory_provider_params_t;
+
+typedef struct umf_cuda_memory_provider_params_t
+    *umf_cuda_memory_provider_params_handle_t;
+
+/// @brief  Create a struct to store parameters of the Cuda memory provider.
+/// @param  hParams [out] handle to the newly created parameters struct.
+/// @return UMF_RESULT_SUCCESS on success or appropriate error code on failure.
+umf_result_t umfCudaMemoryProviderParamsCreate(
```

Comment:
please always write "CUDA" using capital letters as this is an acronym

Reply:
Done